examples/execute-order.js

```diff
@@ -0,0 +1,61 @@
+const bn = require('bn.js')
+const utils = require('./utils')
+
+const privateKey1 = '0x..'
+const privateKey2 = '0x..'
+
+async function init() {
+  const { matic, wallet } = await utils.getMaticClient('testnet', 'v3')
+
+  const token1 = '0x..' // address of token to sell
```

Comment:
would be better to add sample/test token address, amount etc.
User should able to run example code with minimum changes